pif_ingestor/core.py

```diff
@@ -0,0 +1,41 @@
+from .ui import get_cli
+from .manager import run_extension
+from .enrichment import add_tags, add_license, add_contact
+from .uploader import upload
+import os.path
+from pypif import pif
+import logging
+
+
+def main():
```

Comment:
We can add the `if __name__ == "__main__"` pattern, but there needs to be an actual function to point the entrypoint to in the setup.py file.  That entrypoint will cause a script called `pif-ingestor` to be installed in `bin`, which is the preferred way to invoke this.